<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIMO2ViewFile</class>
 <widget class="QWidget" name="UIMO2ViewFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="floatable">
      <bool>true</bool>
     </property>
     <addaction name="actionFOJP"/>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="MO2FileTreeView" name="treeWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="indentation">
       <number>16</number>
      </property>
      <column>
       <property name="text">
        <string notr="true">名前</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>更新日時</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>サイズ</string>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionFOJP">
   <property name="icon">
    <iconset resource="../../Main/src/res.qrc">
     <normaloff>:/res/icon/run.png</normaloff>:/res/icon/run.png</iconset>
   </property>
   <property name="text">
    <string>FOJPテキストに変換する</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MO2FileTreeView</class>
   <extends>QTreeWidget</extends>
   <header>MO2FileTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Main/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
